builds/azure-pipelines/build-release.yml

```diff
@@ -32,12 +32,16 @@ variables:
   LGTM.UploadSnapshot: true
   Codeql.Enabled: true
   Codeql.TSAEnabled: true
+  Codeql.PublishDatabaseLog: true
+  Codeql.Cadence: 0
```

Comment:
The default cadence is 72 hours (meaning codeql will only run every 72 hours) so I am temporarily updating the cadence to 0 to check if codeql is running as expected.